pyproject.toml

```diff
@@ -30,7 +30,8 @@ torch = {version = "^2.2.1", source="torch"}
 torchaudio = {version = "^2.2.1", source="torch"}
 torchvision = {version = "^0.17.1", source="torch"}
 ultralytics = {version = "^8.2.76", source = "PyPI"}
-wandb = {version = "^0.17.5", source = "PyPI"}
+wandb = {version = ">0.17.5", source = "PyPI"}
+yolo_model_development_kit = { git = "https://github.com/Computer-Vision-Team-Amsterdam/YOLO-model-development-kit.git", tag = "v1.0.0" }
```

Comment:
This tag does not exist yet. Reminder to create it (or remove it here so we just take the latest version).